### ISSUE TITLE
"yarn build" with assets hangs.

### ISSUE THREAD

**Issue by j** [nestjs/nest-cli#608]
`nest build` | `nest build myapp` is hanging whenever I have assets.

I've put assets on the global level and the app level and still hangs.  When I remove it, it finishes the build without the assets.

I've tried with globs and without.

## Example nest-cli.json
```json
{
  "collection": "@nestjs/schematics",
  "sourceRoot": "apps/graphql/src",
  "monorepo": true,
  "root": "apps/graphql",
  "compilerOptions": {
    "webpack": true,
    "tsConfigPath": "apps/graphql/tsconfig.app.json"
  },
  "projects": {
    "graphql": {
      "type": "application",
      "root": "apps/graphql",
      "entryFile": "main",
      "sourceRoot": "apps/graphql/src",
      "compilerOptions": {
        "tsConfigPath": "apps/graphql/tsconfig.app.json",
        "assets": [
          "**/.env"
        ]
      }
    }
  }
}
```

**Comment by kamilmysliwiec** [comment]
@Dominic-Preap could you take a look? It seems to be caused by the latest change to the assets manager (migration to chokidar)

**Comment by Dominic-Preap** [comment]
@j Could you please provide a minimal repository which reproduces your issue?

**Comment by kamilmysliwiec** [comment]
Fixed in 7.0.2
